test/e2e-tests.sh

```diff
@@ -120,10 +120,19 @@ kubectl -n "${SYSTEM_NAMESPACE}" patch configmap/config-leader-election --type=m
 add_trap "kubectl get cm config-leader-election -n ${SYSTEM_NAMESPACE} -oyaml | sed '/.*enabledComponents.*/d' | kubectl replace -f -" SIGKILL SIGTERM SIGQUIT
 # Delete HPA to stabilize HA tests
 kubectl -n "${SYSTEM_NAMESPACE}" delete hpa activator
+# Make sure all pods run in leader-elected mode.
+kubectl delete pods --all -n "${SYSTEM_NAMESPACE}"
+add_trap "kubectl delete pods --all -n ${SYSTEM_NAMESPACE}" SIGKILL SIGTERM SIGQUIT
+
+#Try adding timeout to prevent failures
```

Comment:
May be list what we want to kill. Basically HPA and Controller? Rather than all (e.g. 15 activators?)

Reply:
Hmm. We still want to have two activators throughout the test. I plan to modify the HPA for activator have min and max replicas 2. But yeah, we could only restart those that have leader election.